scripts/start_ursim.sh

```diff
@@ -175,44 +217,70 @@ done
 validate_model
 validate_ursim_version
 
+if [ "$URSIM_VERSION" != "latest" ]; then
+  verlte "10.0.0" "$URSIM_VERSION" && ROBOT_SERIES=polyscopex
+fi
+URCAP_STORAGE="${PERSISTENT_BASE}/${ROBOT_SERIES}/urcaps"
+PROGRAM_STORAGE="${PERSISTENT_BASE}/${ROBOT_SERIES}/${ROBOT_MODEL}/programs"
+
+DOCKER_ARGS=""
+
+if [ "$ROBOT_SERIES" == "polyscopex" ]; then
```

Comment:
This could also be added to the ursim validation function, it is also validating the ursim version